packages/mui-base/src/TextareaAutosize/TextareaAutosize.test.tsx

```diff
@@ -458,4 +458,17 @@ describe('<TextareaAutosize />', () => {
       expect(input.style).to.have.property('height', `${lineHeight * 2}px`);
     });
   });
+
+  it('should apply the inline styles using the "style" prop', function test() {
+    if (/jsdom/.test(window.navigator.userAgent)) {
+      this.skip();
+    }
+
+    const { container } = render(<TextareaAutosize style={{ backgroundColor: 'yellow' }} />);
+    const input = container.querySelector<HTMLTextAreaElement>('textarea')!;
```

Comment:
Prefer using the `screen`. We are moving tests as much as possible to rely on global queries. This is purely to keep the test environment simple. Most of the time, we don't need the notion of a container. We render one element at once on the screen.